packages/ruleset/test/schema-description.test.js

```diff
@@ -80,39 +205,6 @@ describe('Spectral rule: schema-description', () => {
       ]);
     });
 
-    it('Named schema with no description', async () => {
```

Comment:
This was no longer a valid negative test once the bug was fixed, because while "Drink" doesn't have a description, it's oneOf list elements do have them.